ppml/README.md

```diff
@@ -0,0 +1,505 @@
+              --jars local:///ppml/trusted-big-data-ml/spark-encrypt-io-0.3.0-SNAPSHOT.jar \
+              local:///ppml/trusted-big-data-ml/work/data/simplequery/spark-encrypt-io-0.3.0-SNAPSHOT.jar \
+              --inputPath /ppml/trusted-big-data-ml/work/data/simplequery/people_encrypted \
+              --outputPath /ppml/trusted-big-data-ml/work/data/simplequery/people_encrypted_output \
+              --inputPartitionNum 8 \
+              --outputPartitionNum 8 \
+              --inputEncryptModeValue AES/CBC/PKCS5Padding \
+              --outputEncryptModeValue AES/CBC/PKCS5Padding \
+              --primaryKeyPath /ppml/trusted-big-data-ml/work/data/simplequery/keys/primaryKey \
+              --dataKeyPath /ppml/trusted-big-data-ml/work/data/simplequery/keys/dataKey \
+              --kmsType EHSMKeyManagementService
+              --kmsServerIP your_ehsm_kms_server_ip \
+              --kmsServerPort your_ehsm_kms_server_port \
+              --ehsmAPPID your_ehsm_kms_appid \
+              --ehsmAPPKEY your_ehsm_kms_appkey
+      ```
+
+
+  3. check runtime status: exit the container or open a new terminal
+
+      To check the logs of the Spark driver, run
+      ```
+      sudo kubectl logs $( sudo kubectl get pod | grep "simplequery.*-driver" -m 1 | cut -d " " -f1 )
+      ```
+      To check the logs of an Spark executor, run
+      ```
+      sudo kubectl logs $( sudo kubectl get pod | grep "simplequery-.*-exec" -m 1 | cut -d " " -f1 )
+      ```
+  
+  4. If you setup [PPML Monitoring](docs/prepare_environment.md#optional-k8s-monitioring-setup), you can check PPML Dashboard to monitor the status in http://kubernetes_master_url:3000
+
+    ![image](https://user-images.githubusercontent.com/61072813/179948818-a2f6844f-0009-49d1-aeac-2e8c5a7ef677.png)
+
+  </details>
+<br />
+
+#### Step 4. Decrypt and Read Result
+When the job is done, you can decrypt and read result of the job. More details in [Decrypt Job Result](./services/kms-utils/docker/README.md#3-enroll-generate-key-encrypt-and-decrypt).
+
+  ```
+  docker exec -i $KMSUTIL_CONTAINER_NAME bash -c "bash /home/entrypoint.sh decrypt $appid $appkey $input_path"
+  ```
+
+### 3.3 More BigDL PPML Examples
+In addition to the above Spark Pi and Python HelloWorld programs running locally, and simplequery application running on the k8s cluster, we also provide other examples including Trusted Data Analysis, Trusted ML, Trusted DL and Trusted FL. You can find these examples in [more examples](./docs/examples.md). 
+
+## 4. Develop your own Big Data & AI applications with BigDL PPML
+
+We provide both Scala and Python APIs to help you develop your own applications with BigDL PPML. 
+
+First thing you need to do is create a `PPMLContext`, who wraps a `SparkSession` and provides read functions to read encrypted data files to plain-text RDD or DataFrame, also provides write functions to save DataFrame to encrypted data files. 
```

Comment:
First you need to create a `PPMLContext`, which wraps `SparkSession` and provides methods to read encrypted data file into plain-text RDD/DataFrame and write DataFrame to encrypted data file. Then you can read & write data through `PPMLContext`.

If you are familiar with Spark, you may find that the usage of `PPMLConext` is very similar to Spark.